src/ClassApplication/Controllers/ValuesController.cs

```diff
@@ -33,5 +33,33 @@ public async Task<ActionResult<string>> GetValues(string id)
             await this.classesContainer.CreateItemAsync<dynamic>(new Object() { });
             return ("Created an object");
         }
+
+        /*------------------------------------------------------------------------------------------------*/

```

Comment:
nit: there is no need for the line since we already have documentation separating it